index.html

```diff
@@ -4,430 +4,336 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ben Omayio</title>
+    <title>Ben Omayio Portfolio</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
+
   <body>
-    <div class="desktop">
-      <main>
-        <header>
-          <a class="logo" href="#">kiboma</a>
-          <ul>
-            <li><a href="#">Portoflio</a> </li>
-            <li><a href="#">About</a> </li>
-            <li><a href="#">Contact</a> </li>
-          </ul>
-        </header>      
-        
+    <header>
+      <nav class="navbar">
+        <a href="#" class="logo">KIBS</a>
+        <ul class="nav_menu">
+          <li class="nav_item">
+            <a href="#portfolio" class="nav_link">Portfolio </a>
+          </li>
+          <li class="nav_item">
+            <a href="#about" class="nav_link">About </a>
+          </li>
+          <li class="nav_item">
+            <a href="#contact" class="nav_link">Contact</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <span class="bar"> </span>
+          <span class="bar"> </span>
+          <span class="bar"> </span>
+        </div>
+      </nav>
+    </header>
+
+    <main>
       <article>
-        <div class="flexbox-container">
-          <div class="flexbox-item flexbox-item-1"> 
-            <h1>I’m Omayio <br /> 
-              Glad to see you!</h1>
+        <div class="row">
+          <div class="intro1 col-12">
+            <h1>
+              I’m Omayio <br />
+              Glad to see you!
+            </h1>
           </div>
-          <div class="flexbox-item flexbox-item-2">           
+          <div class="intro2 col-12">
```

Comment:
- [ Optional ] The headline section is skewed on mobile version. I believe it is due to the desktop image used for mobile. Kindly rectify for better look and feel of your portfolio.
#### Impl
![mobile-headline-section](https://user-images.githubusercontent.com/44057863/167738978-de4356c7-448d-4693-bb7b-f380f2fc1b3b.png)
#### Design
![mobile-headline-section-figma](https://user-images.githubusercontent.com/44057863/167739185-324c8dd5-1dcd-4f9a-963a-fde381764c34.png)